tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -857,6 +1131,34 @@ public function test_evaluate_nested_value() {
 		$this->assertEquals( 'otherPlugin-context-nested', $result );
 	}
 
+	/**
+	 * Tests the `evaluate` method for derived state functions.
+	 *
+	 * @ticket 61037
+	 *
+	 * @covers ::evaluate
+	 * @covers wp_interactivity_state
+	 * @covers wp_interactivity_get_context
+	 */
+	public function test_evaluate_derived_state() {
+		$result = $this->evaluate( 'state.derived' );
+		$this->assertEquals( "Derived state: myPlugin-state\nDerived context: myPlugin-context", $result );
```

Comment:
There's a preference to use `assertSame` where possible (strict equals vs. loose equals):

```suggestion
		$this->assertSame( "Derived state: myPlugin-state\nDerived context: myPlugin-context", $result );
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/6394/commits/3cf4236f8e7c98e6685ed1917572a62c8891d0e5.